GeneratorInterface/Pythia8Interface/plugins/Pythia8Hadronizer.cc

```diff
@@ -29,7 +30,6 @@ using namespace Pythia8;
 // Emission Veto Hooks
 //
 #include "Pythia8Plugins/PowhegHooks.h"
-#include "Pythia8Plugins/PowhegHooksVincia.h"
```

Comment:
This PR backports in 13_2_X also https://github.com/cms-sw/cmssw/pull/42020
However, that PR is supposed to go with https://github.com/cms-sw/cmsdist/pull/8552, which as far as I can see is not backported in 13_2.
Please backport in 13_2 only what is needed for this PR. Updates of Pythia in closed release cycles are delicate and should be either avoided or dealt with by @cms-sw/generators-l2 in a separate and dedicated PR